backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/service/SpecificationDataService.kt

```diff
@@ -72,4 +71,11 @@ class SpecificationDataService @Autowired constructor(
         } catch (exception: PathNotFoundException) {
             null
         }
+
+    fun readId(json: String): String? =
```

Comment:
"extractId"?

Reply:
"getId"? imo most have an idea of what a "getter" does, whereas it wouldn't be immediately clear to me what an "extractor" does on top of that. I'm not a fan of "read", to me something called a "reader" usually involves file I/O?

Reply:
I'm not a fan of "get", because it covers such a wide area. For me, it should only be used in the narrow cases of returning the value if a class field (or similar things). Here, the json parameter is parsed and only a part of it is returned; for me, that is exactly the idea behing "extract".